lua/apg/modules/ghosting.lua

```diff
@@ -47,7 +47,28 @@ function ENT:SetCollisionGroup( group )
 		group = COLLISION_GROUP_INTERACTIVE
 	end
 
-	return APG._SetColGroup( self, group )
+	return APG._SetCollisionGroup( self, group )
+end
+
+ENT._SetColor = ENT._SetColor or ENT.SetColor

```

Comment:
Use... 
```
APG._SetColor = APG._SetColor or ENT.SetColor
```
etc. 

Instead, we want to make sure people can recover the variable from APG. Since it's APG that's detouring it.

Reply:
Done